src/jobs/jobs.service.ts

```
@@ -34,6 +35,9 @@ import { JobStatus } from 'src/feed/enums/job-status.enum';
 
 @Injectable()
 export class JobsService {
+  logger: any;
+  mailerService: any;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)